core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/type/QcType.java

```diff
@@ -252,11 +254,6 @@ public String toString() {
     return sb.toString();
   }
 
-  @Override
-  public long getId() {
-    return qcTypeId;
-  }
-
   @Override
   public String getAlias() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Update this method so that its implementation is not identical to "getName" on line 97. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4144)